orchestrator/test_runner/src/main.rs

```diff
@@ -170,6 +195,20 @@ pub async fn main() {
         return;
     }
 
+    // if we detect this env var, run the relayer until it terminates for whatever reason
+    if should_setup_and_run_ibc_relayer() {
+        info!("test-runner in ibc relayer deploying mode: init hermes, create ibc channel, start hermes");
+        let mut hermes_base = Command::new("hermes");
+        let hermes_base = hermes_base.arg("-c").arg(HERMES_CONFIG);
+        let _res = setup_relayer_keys(keys[0].clone(), ibc_phrases[0].clone()).unwrap();
+        create_ibc_channel(hermes_base);
+        // Need to recreate hermes command: start can't come after create channel, no clone options
+        let mut hermes_base = Command::new("hermes");
+        let hermes_base = hermes_base.arg("-c").arg(HERMES_CONFIG);
+        run_ibc_relayer(hermes_base, true); // likely will not return from here, just keep running
```

Comment:
we could spawn the relayer in a new rust thread to keep it from blocking? I don't think it matters in this case though.